.emacs.d/lisp/code/languages/ruby.el

```diff
@@ -0,0 +1,21 @@
+;; Set ruby-indent-line to global-tab-width
+(setq-default ruby-indent-line global-tab-width)
+
+;; Provide advice to Smart Tabs about how to handle Ruby's special
+;; indentation variables.
+(smart-tabs-advice ruby-indent-line ruby-indent-level)
+
+;; Apply Ruby language support to Smart Tabs.
+(smart-tabs-insinuate 'ruby)
+
+;; Use tabs mode in Ruby by default.
+(setf ruby-indent-tabs-mode t)
```

Comment:
Yes.  For instances where we have "special" (i.e. arrogant) language modes which disobey `indent-tabs-mode` in favor of their own indent-tabs-mode variables, we need to specify these.